Formula/embulk.rb

```diff
@@ -4,8 +4,8 @@ class Embulk < Formula
   # https://www.embulk.org/articles/2020/03/13/embulk-v0.10.html
   # v0.10.* is a "development" series, not for your production use.
   # In your production, keep using v0.9.* stable series.
```

Comment:
Mmm, actually, there must be something wrong.

Reply:
exactly, I created [an issue](https://github.com/embulk/embulk/issues/1613) for discussing whether this upgrade is really needed.